static/js/tabbed-content.js

```diff
@@ -183,9 +183,11 @@
       tabContent.forEach((content) => {
         if (tabElement === tab) {
           tabElement.setAttribute("aria-selected", true);
+          tabElement.classList.add("is-active");
```

Comment:
What purpose does adding this class serves, this might not have relevance to current issue.